packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -56,23 +70,36 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
             this._paypalCommerceSdk = await this._paypalScriptLoader.getPayPalSDK(paymentMethod, currencyCode, paypalcommerce.initializesOnCheckoutPage);
         }
 
-        this._renderButton(containerId, methodId, paypalcommerce);
+        this._renderButton(containerId, methodId, paypalcommerce, isHostedCheckoutEnabled);
```

Comment:
You don't need to pass isHostedCheckoutEnabled here. Because you can get it from payment method inside the method.

Reply:
removed